compiler/rustc_span/src/lib.rs

```diff
@@ -124,7 +133,7 @@ pub enum RealFileName {
 
 impl RealFileName {
     /// Returns the path suitable for reading from the file system on the local host.
-    /// Avoid embedding this in build artifacts; see `stable_name` for that.
+    /// Avoid embedding this in build artifacts; see `stable_name()` for that.
```

Comment:
Is there a fixed convention for this? The old comment was referring to a function by name, which seems more correct. If there isn’t a well understood convention I could imagine someone else coming along and reverting these changes as part of similar comment tidying exercise.

Reply:
i don't think that we have a fixed convention for this

> If there isn’t a well understood convention I could imagine someone else coming along and making the opposite changes as part of a similar comment tidying exercise.

yeah :shrug: i mostly just don't mind either way so imo asking people to remove/add changes like this seems slightly too pedantic to me.